src/PowerSystems.jl

```diff
@@ -206,6 +207,7 @@ export get_dynamic_components
 
 export solve_powerflow!
 export solve_powerflow
+export is_connected
```

Comment:
should we rename this has_islands?

Reply:
I think of the "island" terminology as an operational situation rather than a network connectivity issue. I prefer "connected" or "is_connected". But, I could be persuaded otherwise.

Reply:
Is connected seems to imply singular, doesn't represent that there is a system that is continuously connected.

Reply:
maybe something more like `PM.check_connectivity()`

Reply:
we can use validate_connectivity() since in PSY check is meant to throw an error and validate return true/false